provisioning/roles/php/tasks/main.yml

```diff
@@ -75,10 +75,10 @@
   ignore_errors: true
   when: ansible_os_family == "Debian" and use_xdebug == true
 
-- name: Enable XDebug
-  command: php5enmod xdebug
+- name: Disable XDebug
+  command: php5dismod xdebug
   ignore_errors: true
-  when: ansible_os_family == "Debian" and use_xdebug == true
+  when: ansible_os_family == "Debian" and use_xdebug == true and activate_xdebug == false
```

Comment:
Add description to https://github.com/netgen/vagrant-ansible-ezpublish/blob/disable_xdebug_by_default/provisioning/roles/php/README.md under variables.

Consider explicitly stating it here https://github.com/netgen/vagrant-ansible-ezpublish/blob/disable_xdebug_by_default/provisioning/vagrant.yml#L9 so it is easier to change for people that encounter the setup for the first time.